sormas-backend/src/main/java/de/symeda/sormas/backend/vaccination/VaccinationService.java

```diff
@@ -223,25 +225,31 @@ public boolean isVaccinationRelevant(Contact contact, Vaccination vaccination) {
 	 * @return true when the vaccination is relevant, false otherwise
 	 */
 	public boolean isVaccinationRelevant(Event event, Vaccination vaccination) {
-		if (event.getStartDate() != null) {
-			return isVaccinationRelevant(vaccination, event.getStartDate(), null);
-
-		} else {
-			return isVaccinationRelevant(vaccination, event.getEndDate(), event.getReportDateTime());
-		}
-
+		return isVaccinationRelevant(vaccination, event.getStartDate(), event.getEndDate(), event.getReportDateTime());
 	}
 
 	/*
 	 * HEADS UP! If you make changes here, you most probably also need to update the database queries in
 	 * ContactService.updateVaccinationStatuses(...) and
 	 * EventParticipantService.updateVaccinationStatuses(...).
 	 */
-	private boolean isVaccinationRelevant(Vaccination vaccination, Date primaryDate, Date fallbackDate) {
+	public boolean isVaccinationRelevant(Vaccination vaccination, Date... relevanceFilterDates) {
```

Comment:
Why is this method now public? Not used outside that Service

Reply:
Was used from outside the class in an intermediary step of the refactoring.
Fixed.